dips/dip-158.md

```diff
@@ -0,0 +1,794 @@
+ 
+This DIP extends [DIP-1](https://dip.diem.com/dip-1/) which specifies P2P payment flows (Peer To Peer).
+
+This DIP replaces the Auth/Capture section in [DIP-8](https://dip.diem.com/dip-8/#authcapture).
+
+This DIP does NOT include payment flows which require funds pull pre-approval such as subscription and recurring payments. 
+
+This DIP makes use of the `result` field added to the `CommandResponseObject` as described in DIP-161
+
+---
+# Abstract / Motivation
+
+
+This DIP is an extension of the Off-Chain Protocol to support more advanced payment functionality - particularly eCommerce P2M payments. 
+
+## How P2M Payments Differ from P2P Transfers 
+There are several key differences between the process of paying a merchant (P2M) and a simple P2P transfer:
+
+|      |P2M|P2P|
+|------|---|---|
+|**Payer Data**|Buyer details are required for **every** payment / purchase in order to perform risk checks and apply AML policies|Required only for transfers for which Travel Rule applies. i.e., over 1000 USD|
+|**Payment Owner**|The merchant, i.e. the receiver of funds, "drives" the process and controls the type of payment and its timing. For example, the merchant will decide when to capture the previously authorized funds and whether to capture the entire payment amount or only partial amount |The payer, i.e. the sender of funds, determines the nature of the transfer|
+|**Reconciliation**|The merchant requires a way to correlate the transfer of funds to a payment or payment request for reconciliation purposes|No reconciliation|
+|
+
+## P2M Payment Types
+There are two types of P2M payments described in this document: 
+
+1. **Charge**. Charge payment stipulates an immediate capture of funds. This means that a successful off-chain negotiation for such payment will result in funds being captured 
+2. **Auth / Capture**. Auth / Capture payment, as the name suggests, means that the payment is divided to two steps: Authorization and Capture. Successful Authorization implies that the funds are locked so they can later be captured
+
+
+---
+# Glossary
+
+For the sake of simplicity, we use the following terms:
+* "Sender", "Payer", "Wallet", "Buyer VASP" or "User VASP" - The user's wallet. It is the VASP representing the customer/buyer. This is the VASP that will send the funds and eventually will submit the on-chain transaction to the blockchain
+* "Receiver", "Receiving VASP" or "Merchant VASP" - The VASP representing the merchant which receives the funds (transaction)
+* "Buyer", "Customer" or "User" - The person who wishes to pay the merchant for goods or services
+
+
+
+---
+# Pre-requisite: Sharing Payment Identifier and Receiver Address
+
+A pre-requisite for the off-chain negotiations is that both parties (sending and receiving VASPs) have shared a common payment identifier and the receiver address. This DIP specifies a couple of examples for exchanging identifiers between VASPs, however,  there could be others.
```

Comment:
I would add to "payment identifier" (or replace it) a link to DIP-1 terminology "transaction reference-ID"

Reply:
I added an item to the Glossary section about payment identifier which states all the synonyms and contains a link to DIP terminology.